src/engine/loopingcontrol.cpp

```diff
@@ -53,23 +53,45 @@ LoopingControl::LoopingControl(QString group,
             Qt::DirectConnection);
     m_pLoopInButton->set(0);
 
+    m_pLoopInGotoButton = new ControlPushButton(ConfigKey(group, "loop_in_goto"));
+    connect(m_pLoopInGotoButton, SIGNAL(valueChanged(double)),
+            this, SLOT(slotLoopInGoto(double)));
+
     m_pLoopOutButton = new ControlPushButton(ConfigKey(group, "loop_out"));
     connect(m_pLoopOutButton, SIGNAL(valueChanged(double)),
             this, SLOT(slotLoopOut(double)),
             Qt::DirectConnection);
     m_pLoopOutButton->set(0);
 
+    m_pLoopOutGotoButton = new ControlPushButton(ConfigKey(group, "loop_out_goto"));
+    connect(m_pLoopOutGotoButton, SIGNAL(valueChanged(double)),
+            this, SLOT(slotLoopOutGoto(double)));
+
+
     m_pLoopExitButton = new ControlPushButton(ConfigKey(group, "loop_exit"));
     connect(m_pLoopExitButton, SIGNAL(valueChanged(double)),
             this, SLOT(slotLoopExit(double)),
             Qt::DirectConnection);
     m_pLoopExitButton->set(0);
 
-    m_pReloopExitButton = new ControlPushButton(ConfigKey(group, "reloop_exit"));
-    connect(m_pReloopExitButton, SIGNAL(valueChanged(double)),
-            this, SLOT(slotReloopExit(double)),
```

Comment:
reloop_toggle is confusing the other way round, since the hardware button is labeled "Reloop/Exit" 
Now I am in favor to keep the old name instead of clutter the CO interface by an alias. 
If we really need a new name "reloop_or_exit" might do.